packages/components/src/components/tag/tag.ts

```diff
@@ -0,0 +1,163 @@
+  focus(options?: FocusOptions) {
+    this.tag.focus(options);
+  }
+
+  /** Removes focus from the tag. */
+  blur() {
+    this.tag.blur();
+  }
+
+  render() {
+    const isLink = this.isLink();
+    const tag = isLink ? literal`a` : literal`button`;
+
+    const slots = {
+      label: this.hasSlotController.test('[default]'),
+      'removable-indicator': this.hasSlotController.test('removable-indicator')
+    };
+
+    /* eslint-disable lit/no-invalid-html */
+    /* eslint-disable lit/binding-positions */
+    return html`
+      <${tag}
+        part='base'
+        type=${ifDefined(isLink ? undefined : 'button')}
+        href=${ifDefined(isLink ? this.href : undefined)}
+        rel=${ifDefined(isLink && this.target ? 'noreferrer noopener' : undefined)}
+        target=${ifDefined(isLink ? this.target : undefined)}
+        download=${ifDefined(isLink ? this.download : undefined)}
+        ?disabled=${ifDefined(isLink ? undefined : this.disabled)}
+        aria-disabled=${this.disabled ? 'true' : 'false'}
+        tabindex=${this.disabled ? '-1' : '0'}
+        @blur=${this.handleBlur}
+        @focus=${this.handleFocus}
+        class=${cx(
+          /* basic styles of the wrapper */
+          'inline-flex border box-border rounded-full items-center group leading-none whitespace-nowrap focus-visible:focus-outline',
+          {
+            /* sizes, fonts */
+            lg: 'h-8 text-base gap-2',
+            sm: 'h-6 text-sm gap-1'
+          }[this.size],
+          {
+            /* padding */
+            lg: !this.removable ? 'px-4 py-2' : 'pl-4 pr-3 py-2',
+            sm: !this.removable ? 'px-3 py-[5px]' : 'pl-3 pr-2 py-2'
+          }[this.size],
+          /* colors */
+          !this.selected
+            ? 'border-primary text-primary hover:border-primary-500 hover:bg-neutral-100 hover:text-primary-500  disabled:border-neutral-500 disabled:text-neutral-500'
+            : 'bg-primary border-primary text-white hover:bg-primary-500 hover:border-primary-500 disabled:bg-neutral-500 disabled:border-neutral-500',
+          this.disabled && !isLink && 'cursor-not-allowed'
+        )}
+      >
+        <slot part='content'></slot>
+        <slot part='removable-indicator' name='removable-indicator'></slot>
+        <div part='removable-indicator' class=${cx(
+          'relative flex flex-col justify-center',
+          {
+            lg: 'h-4 w-4',
+            sm: 'h-3 w-3'
+          }[this.size],
+          !this.removable && 'hidden',
+          slots['removable-indicator'] && 'hidden'
+        )}>
+          <div class=${cx(
+            'absolute w-full h-[1px]  -rotate-45',
+            !this.selected ? 'bg-primary group-hover:bg-primary-500 group-disabled:bg-neutral-500' : 'bg-white'
+          )}></div>
+          <div class=${cx(
+            'absolute w-full h-[1px]  rotate-45',
+            !this.selected ? 'bg-primary group-hover:bg-primary-500 group-disabled:bg-neutral-500' : 'bg-white'
+          )}></div>
```

Comment:
1. There should only be one single part.
2. Please add the X to the system icons instead of building it yourself via CSS.
3. The icon should be the fallback of the slot, therefore this shoudl be something like
```html
<slot part='removable-indicator' name='removable-indicator'>
<sd-icon library="system" icon="x"></sd-icon>
</slot>
```
4. The slot of course should only show up if removable is activated.

Please ask @coraliefeil what the final name of the `x` will be in future.

Reply:
5. The hovering etc. will be handled via text color as soon as you add an sd-icon. So no group etc. needed.

Reply:
> 5. The hovering etc. will be handled via text color as soon as you add an sd-icon. So no group etc. needed.

Yeah, using a SVG makes it way easier actually.